custom_components/proxmoxve/config_flow.py

```diff
@@ -287,6 +288,12 @@ async def async_step_change_expose(
                                 CONF_DISKS_ENABLE, True
                             ),
                         ): selector.BooleanSelector(),
+                        vol.Optional(
+                            CONF_ZFS_ENABLE,
+                            default=self.config_entry.options.get(
+                                CONF_ZFS_ENABLE, True
+                            ),
+                        ): selector.BooleanSelector(),
```

Comment:
We can keep only the "Enable disks" option and it will also work for ZFS.

The option exists only to avoid interfering with disk hibernation.